Dockerfile-node

```
@@ -1,6 +1,10 @@
 FROM node:16.20.1
```

```suggestion
FROM node:18.18.2
```
We updated this recently in docker-compose to match Heroku.